index.html

```diff
@@ -229,7 +229,7 @@ <h3 class="codeused  "> Skills
         </div>
     </section>
 
-    <footer class="contact-menu">
+    <article class="contact-menu">
```

Comment:
- [ ] As the previous code reviewer suggested, the `section` tag would be more appropriate in this case for HTML semantic reasons. Kindly change that. Please, [read this article](https://www.w3schools.com/tags/tag_article.asp) for more information about when to use the `article` tag.